test/controllers/returns/PlasticExportedByAnotherBusinessControllerSpec.scala

```diff
@@ -137,11 +137,6 @@ class PlasticExportedByAnotherBusinessControllerSpec
     }
 
     "redirect to the account page if cannot calculate total plastic" in {
-      val userAnswer = UserAnswers("123")
```

Comment:
going to get conflicts in the returns files as this doesnt relate to this pr, and has been done in marcys pr

Reply:
not sure what you mean, just removed this as it was not used